specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json

```diff
@@ -4043,6 +4616,90 @@
         }
       ]
     },
+    "SqlStoredProcedureGetResults": {
+      "description": "An Azure Cosmos DB storedProcedure.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties of an Azure Cosmos DB storedProcedure",
+          "$ref": "#/definitions/SqlStoredProcedureGetProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ARMResourceProperties"
+        }
+      ]
+    },
+    "SqlStoredProcedureGetProperties": {
+      "description": "The properties of an Azure Cosmos DB userDefinedFunction",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SqlUserDefinedFunctionResource"
```

Comment:
SqlStoredProcedureGetProperties shouldn't contain SqlUserDefinedFunctionResource. The documentation for this will be generated referencing User Defined Function descriptions.
Create a new type definition for the properties of the stored proc.